assets/quick-order-list.js

```diff
@@ -79,11 +79,11 @@ class QuickOrderList extends HTMLElement {
         type: `${this.stickyHeaderElement.getAttribute('data-sticky-type')}`
       };
     }
-
-    this.totalBarPosition = window.innerHeight - this.querySelector('.quick-order-list__total').offsetHeight;
+    this.totalBar = this.querySelector('.quick-order-list__total');
+    this.totalBarPosition = this.totalBar ? window.innerHeight - this.totalBar.offsetHeight: null;
```

Comment:
why is the total bar position set to `null` and not `0` ? 

The only place we use it is `const totalBarCrossesInput = inputBottomBorder > this.totalBarPosition;` so is it doing `> null` check?